fw-child/resources/js/child-functions.js

```diff
@@ -189,6 +189,38 @@ const $ = jQuery;
       }
     }
 
+    // SMOOTH SCROLL
+
+    let glossaryScrollOffset = 0;
```

Comment:
Probably something like `smoothScrollOffset` could be more generalized, since the smooth scrolling could be used on other pages.